Source/WebCore/rendering/HitTestResult.cpp

```diff
@@ -528,6 +529,48 @@ void HitTestResult::enterFullscreenForVideo() const
 #endif
 }
 
+bool HitTestResult::videoHasSubtitles() const
+{
+#if ENABLE(VIDEO)
+    if (HTMLMediaElement* mediaElement = this->mediaElement()) {
+        if (auto* textTracks = mediaElement->textTracks())
+            return textTracks->length();
```

Comment:
I think we only want to show the item if there is a caption or subtitles track with cues. This will show the item if an element has a metadata track, a chapter track, an empty captions track, etc.